src/Service.Host/client/src/components/PurchaseOrders/PurchaseOrderUtility.js

```diff
@@ -152,6 +153,10 @@ function PurchaseOrderUtility({ creating }) {
         itemSelectorHelpers.getItemLoading(state[purchaseOrderDeliveries.item])
     );
 
+    const requestErrors = useSelector(reduxState =>
+        getRequestErrors(reduxState)?.filter(error => error.type !== 'FETCH_ERROR')
+    );
+
```

Comment:
out of interest, what so-called 'requestError' does this catch? i.e. which ones aren't error.type == 'FETCH_ERROR'